src/html/ReqMgr/templates/doc.tmpl

```diff
@@ -86,8 +103,17 @@ function Activate(tag, hidetags) {
     }
     ShowTag('edit-'+tag);
 }
+function UpdateTaskParams() {
+    var parameters = \$('#taskParams').val();
+    ajaxRequest('$base/data/splitting/$rid', parameters, 'POST');
```

Comment:
Ok, I put it back.

On  0, ticoann <notifications@github.com> wrote:
> > @@ -86,8 +103,17 @@ function Activate(tag, hidetags) {
> >      }
> >      ShowTag('edit-'+tag);
> >  }
> > +function UpdateTaskParams() {
> > +    var parameters = \$('#taskParams').val();
> > +    ajaxRequest('$base/data/splitting/$rid', parameters, 'POST');
> 
> HI Valentin,  ajaxRequest will stringify. However 
> ``` javascritpt
> var parameters = \$('#taskParams').val();
> ```
> parameters here is str not javascript object.  so it is stringifying  over string not the javascript object.
> On the server side decode json instead of it is getting json object it gets the string back.
> 
> https://github.com/dmwm/WMCore/pull/6596/files#diff-f192bc1cb387d886ce13140d344d5373R152
> 
> I changed the code reqmgr2-dev.cern.ch  vm  (add JSON.parse(parameter)), it seems it is working now. 
> https://reqmgr2-dev.cern.ch/reqmgr2/fetch?rid=valya_160219_150502_7975
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/dmwm/WMCore/pull/6605/files#r53541690